frontend/src/scenes/App.tsx

```diff
@@ -117,6 +118,7 @@ export function App(): JSX.Element | null {
                     {!sceneConfig.hideTopNav && <TopNavigation />}
                     {scene ? (
                         <Layout.Content className="main-app-content" data-attr="layout-content">
+                            {user.is_async_event_action_mapping_enabled && <AsyncActionWarning />}
```

Comment:
I wasn't suggesting always showing this, just when you are creating a webhook (and maybe when you are looking at an action).